backend/src/core/utils/datetime.py

```diff
@@ -21,4 +40,23 @@ def to_bkk_string(cls, dt: datetime) -> str:
     @classmethod
     def from_string(cls, date_str: str) -> datetime:
         """Convert a standardized string back to a datetime object."""
-        return cls.BKK_TZ.localize(datetime.strptime(date_str, cls.FORMAT))
\ No newline at end of file
+        return cls.BKK_TZ.localize(datetime.strptime(date_str, cls.FORMAT))
+
+# Custom SQLAlchemy type for Bangkok timezone dates
+class BkkDateTime(TypeDecorator):
```

Comment:
Why don't we just use UTC???????????????

Reply:
u can convert utc to thai time string

Reply:
date time should be store as utc. we aren't making any promotion that need to happen at localtime.